src/common/constants/dmca.json

```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_urls": ["cracked-games/ltxziiz9", "bleh773/2ryjej-on-off"]
```

Comment:
Can you name it `posts` and put them on multiple line?

Reply:
yup just did